achievements/account_pages/views.py

```diff
@@ -0,0 +1,27 @@
+from django.contrib.auth.mixins import PermissionRequiredMixin
+from django.views.generic import TemplateView
+
+from membership_file.util import MembershipRequiredMixin
+from user_interaction.account_pages.mixins import AccountTabsMixin
+
+from achievements.models import Category
+from achievements.serializers import CategorySerializer, AchievementSortType
+
+
+class AchievementAccountView(MembershipRequiredMixin, AccountTabsMixin, TemplateView):
```

Comment:
Seems like the test for this view was removed (see `achievements/tests/tests.py`). I don't really mind though given that this module should be refactored anyways.

Reply:
I don't see why membership is required for this view? My intentions with this Mixin were to limit view accessibility to members where it would not actually be possible for non-members to go regardless of permissions (e.g. editing membership information, which would not be possible for non-members).
In my opinion, things that are theoretically possible by non-members should be restricted using permissions if they should not be allowed to go there. I don't see why we would need to restrict a access to user's own earned achievements though (this is currently not the case either).

Reply:
1) Coherence, we do not support this page code-wise for non-members so we shouldn't allow this page linked like this
2) We should ideally have achievements connected to members, not users. In fact, I thought it was connected to members, not users.